fix(paginationcomponent): hide label when pagination component is hidden

NemirStephan opened the pull request:
# Description

The Pagination component does not render when the totalItems is <= itemsPerPage. However, the label
is still being show. This fixes that issue

Fixes #1166

Screenshot of existing issue

<img width="896" alt="Screenshot 2023-03-17 at 11 14 12" src="https://user-images.githubusercontent.com/22469863/225890796-af97ee6a-302f-4f18-92b7-073659c1af47.png">
 (if appropriate provide a quick screen grab of the changes)

# Checklist:

- [x] The commit messages follow the [convention for this project](./blob/master/docs/CONTRIBUTING.md#conventional-commits)
- [x] I have provided an adequate amount of test coverage
- [ ] I have added the functionality to the [test app](./blob/master/docs/CONTRIBUTING.md#build-test-application)
- [ ] I have provided a story in storybook to document the changes
- [ ] I have added the documentation
- [ ] I have added any new public feature modules to public-api.ts

pauleustice commented on projects/canopy/src/lib/pagination/pagination.component.html:
```diff
@@ -1,40 +1,43 @@
-</ul>
-<div class="lg-pagination__label lg-font-size-0-8" >{{label}}</div>
+<ng-container *ngIf="pages.length > 1">
+  <ul *ngIf="pages.length > 1">
```
I guess this if can be removed right?

NemirStephan replied:
done

pauleustice commented:
Maybe it's just me, but I like the label being there! 🤷 😄

elenagarrone commented:
Could you please update the context of the commit to only be `pagination` please? e.g.:
```
fix(pagination): hide label when pagination component is hidden
```

elenagarrone commented on projects/canopy/src/lib/pagination/pagination.component.html:
```diff
@@ -1,40 +1,43 @@
+  </ul>
+  <div class="lg-pagination__label lg-font-size-0-8" >{{label}}</div>
+</ng-container>
+
```
I think we can remove this extra blank line?

NemirStephan replied:
done, but I have a feeling, if Walter reviews it he might ask for it to be put back :)

NemirStephan commented:
I'd be happy to leave it in if people prefer. I can check with UX to see what they think

elenagarrone commented:
> @pauleustice @elenagarrone I can leave the label there if people prefer it. I'll check with UX to see what they think

Worth checking with the UX designer 👍

NemirStephan commented:
> 

I checked with Fenton and although he's happy to leave it as it is, he said its probably better to hide it too

NemirStephan commented:
> 



> fix(pagination): hide label when pagination component is hidden

I've fixed this and rebased so there is only 1 commit now. Hope thats ok
